torch/onnx/symbolic_opset9.py

```diff
@@ -6629,3 +6630,23 @@ def onnx_placeholder(g: jit_utils.GraphContext, *inputs, **attrs):
     env = g.env
 
     return torch._C._jit_onnx_convert_pattern_from_subblock(block, node, env)
+
+
+@_onnx_symbolic("aten::_conj")
+@_onnx_symbolic("aten::conj_physical")
+@_onnx_symbolic("aten::resolve_conj")
+@_onnx_symbolic("aten::resolve_neg")
+@_beartype.beartype
+def unsupported_complex_operators(g: jit_utils.GraphContext, input: _C.Value):
+    # ONNX does not have operators to *directly* manipulate real/imaginary components
+    # However, a few torch APIs (e.g. .tolist()) use complex operations when input is real,
+    # which results in failures due to missing operators for complex numbers
+
+    # While `aten::_conj` and `aten::conj_phisical` raise exception when input is complex
+    if symbolic_helper.is_complex_value(input):
+        return symbolic_helper._onnx_unsupported(
+            "aten::_conj, aten::conj_physical, aten::resolve_conj, aten::resolve_neg",
+            input,
+        )
+    # `aten::resolve_conj` and `aten:: resolve_neg` can safely be implemented as no-op
+    return input
```

Comment:
Looks like the logic doesn't match the comment? I would break out resolve_conj and resolve_neg to a separate function